lib/users/users.yml

```diff
@@ -10,7 +10,14 @@ attributes:
     id: password
     name: Password
     description: Make it secure!
-  - type: text
-    id: level
-    name: Access Level
-    description: (make multi-select plugin w/selection of CTs for `content-creator`) CHOOSE one - superuser/admin/content-creator
+  - type: select
+    id: role
+    name: Authorization role
+    label: Select authorization level
+    inputs:
```

Comment:
These need to pull from the config. We're probably going to need to make this a `.js` file, not a `.yml` file to accomidate that